Automattic/wp-calypso
# Stats: Deprecate feature flag "stats/new-main-chart" within shared components

#### Proposed Changes

* Introduce prop isLegacy to component `StatsNavigation`.
* Refactor component `StatsDatePicker` with the short title.
* Refactor component `StatsPeriodNavigation` with the new structure.
* Refactor component `Delta` with new suffix styles.
* Refactor modernized `StatsNavigation` styles.
* Refactor modernized `FormattedHeader` styles.
* Remove the modernized page header styles file.

#### Testing Instructions



> Note: The instructions to show the Store and Ads pages can be found at https://github.com/Automattic/wp-calypso/pull/69661 and https://github.com/Automattic/wp-calypso/pull/69743.

* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Traffic`, `Insights`, `Store`, `Ads`, `Overview (/stats/day)`, and `Google My Business (/google-my-business/stats/${your-site})` pages.
* Ensure the styles are the same as the previous ones.
* a12s: Since the `Google My Business` tab would be hard to reproduce, we can `su` into a local instance with details from `p1670518452086579-slack-C0438NHCLSY`.

#### Pre-merge Checklist





Related to #70209
